src/data/results.html

```diff
@@ -17,8 +17,8 @@
   </head>
   <body>
 
-    <button id='monthDown' onClick="changeMonthDown();">Previous</button>
-    <button id='monthUp' onClick="changeMonthUp();">Next</button>
+    <button id='monthDown'>Previous</button>
+    <button id='monthUp'>Next</button>
```

Comment:
inline JS is not permitted in web extensions -- there is a default CSP, that _can_ be changed, but we still can't enable inline JS (source: https://developer.mozilla.org/fr/Add-ons/WebExtensions/manifest.json/content_security_policy)